compiler/src/iree/compiler/Codegen/LLVMCPU/test/hal_target_device_attributes.mlir

```diff
@@ -0,0 +1,33 @@
+// This test aims to check default HAL properties for LLVM CPU target, and
```

Comment:
You need to add this test to compiler/src/iree/compiler/Codegen/LLVMCPU/test/CMakeLists.txt. Since it's autogenerated from Bazel, you must add it to BUILD.bazel and invoke the bazel_to_cmake converter.py.

Reply:
Done, thanks for the hint!